tendrl/gluster_integration/sds_sync/georep_details.py

```diff
@@ -36,6 +38,56 @@ def save_georep_details(volumes, index):
                         index, pair_index)
                 ].replace("/", "_")
             )
+
+            try:
+                fetched_pair_status = NS._int.client.read(
+                    "clusters/%s/Volumes/%s/GeoRepSessions/"
+                    "%s/pairs/%s/status" % (
+                        NS.tendrl_context.integration_id,
+                        volumes['volume%s.id' % index],
+                        session_id,
+                        pair_name
+                    )
+                ).value
+                pair_status = volumes[
+                    'volume%s.pair%s.status' % (index, pair_index)
+                ]
+                if fetched_pair_status != pair_status and \
+                    pair_status.lower() == 'faulty':
```

Comment:
the if block below looks very similar, can we club it to single block by having `pair_status.lower() in ['faulty','active', 'passive']:` and having place holder for new status

Reply:
so the logic is like in case current status is `faulty`, the previous could be anything. But if the current status is `active`/`passive`, the old value we look for `faulty` only so I kept as separate conditions.

Reply:
Ack